packages/admin-stories/src/docs/components/Table/FilterBar/stories/Default.stories.tsx

```diff
@@ -0,0 +1,207 @@
+import {
+    Field,
+    FilterBar,
+    FilterBarMoreFilters,
+    FilterBarPopoverFilter,
+    FinalFormInput,
+    FinalFormRangeInput,
+    FinalFormSearchTextField,
+    FinalFormSwitch,
+    Table,
+    TableFilterFinalForm,
+    useTableQueryFilter,
+} from "@comet/admin";
+import { FinalFormReactSelectStaticOptions } from "@comet/admin-react-select";
+import { Box, Divider, FormControlLabel, Typography } from "@material-ui/core";
+import { storiesOf } from "@storybook/react";
+import faker from "faker";
+import * as React from "react";
+
+interface ColorFilterFieldProps {
+    colors: string[];
+}
+
+const ColorFilterField: React.FC<ColorFilterFieldProps> = ({ colors }) => {
+    const options = colors
+        .filter((color, index, colorsArray) => colorsArray.indexOf(color) == index) //filter colorsArray to only have unique values as select options
+        .map((color) => {
+            return { value: color, label: color };
+        });
+
+    return <Field name="color" type="text" component={FinalFormReactSelectStaticOptions} fullWidth options={options} />;
```

Comment:
We're trying to get rid of react-select. Therefore we shouldn't use it in new stories, in my opinion

Reply:
i will change it to the new Select :)